gui/wxpython/history/tree.py

```diff
@@ -76,16 +76,17 @@ def dayToLabel(self, day):
         if day == OLD_DATE:
             return _("No time info")
 
+        month_name = day.strftime("%B")
         if day == current_date:
-            return "{:%B %-d} (today)".format(day)
+            return "{0} {1} (today)".format(month_name, day.day)
         elif day == current_date - datetime.timedelta(days=1):
-            return "{:%B %-d} (yesterday)".format(day)
+            return "{0} {1} (yesterday)".format(month_name, day.day)
         elif day >= (current_date - datetime.timedelta(days=current_date.weekday())):
-            return "{:%B %-d} (this week)".format(day)
+            return "{0} {1} (this week)".format(month_name, day.day)
         elif day.year == current_date.year:
-            return "{:%B %-d}".format(day)
+            return "{0} {1}".format(month_name, day.day)
         else:
-            return "{:%B %-d, %Y}".format(day)
+            return "{0} {1}, {2}".format(month_name, day.day, day.year)
```

Comment:
This assumes English word order for dates right?

Reply:
Yep, the order in American English month – day – year.

Reply:
Is it expected to be localizable?

Reply:
I did not think about it.. it could be, I added _ in front of strings. Thanks for pointing out.

Reply:
> Yep, the order in American English month – day – year.

Is that given or you it be(come) ISO style?